include/stellar/io/import_sequence.hpp

```diff
@@ -0,0 +1,163 @@
+#pragma once
+
+#include <seqan/seq_io.h>
+
+namespace stellar
+{
+
+using namespace seqan2;
+
+template <typename TId>
+inline bool
+_checkUniqueId(std::set<TId> & uniqueIds, TId const & id)
+{
+    TId shortId;
+    typedef typename Iterator<TId const>::Type TIterator;
+
+    TIterator it = begin(id);
+    TIterator itEnd = end(id);
+
+    // (cut at first whitespace)
+    while (it != itEnd && *it > ' ')
+    {
+        appendValue(shortId, *it);
+        ++it;
+    }
+
+    if (uniqueIds.count(shortId) == 0)
```

Comment:
c++20 now has `std::set::contains` as a function
```suggestion
    if (!uniqueIds.contains(shortId))
```